documentation/source/getting_started/quick_install.rst

```diff
@@ -23,6 +23,10 @@ Perform this procedure to run OpenEBS operator. Use the following commands at th
 
    1.  kubectl create/apply -f openebs-operator.yaml
    2.  kubectl create/apply -f openebs-storageclasses.yaml
+   
+**Installation using helm:**
```

Comment:
@SiddharthMahapatra12 -- the instructions need to be explicitly mentioned here.. WE can't link to the github docs page.

Reply:
Sure sir,
will modify accordingly with a new PR.